programs/bpf_loader/src/lib.rs

```diff
@@ -83,6 +82,8 @@ pub fn create_executor(
         max_call_depth: compute_budget.max_call_depth,
         stack_frame_size: compute_budget.stack_frame_size,
         enable_instruction_tracing: log_enabled!(Trace),
+        verify_mul64_imm_nonzero: !invoke_context
+            .is_feature_active(&stop_verify_mul64_imm_nonzero::id()), // TODO: Feature gate and then remove me
         ..Config::default()
```

Comment:
Are the stack frame gaps disabled by default now?  If so they should be featurized

Reply:
Is the Config's `instruction_meter_checkpoint_distance` default the same as v0.2.14?  If not, also should be feature gated

Reply:
Still enabled by default: `enable_stack_frame_gaps: true`
https://github.com/solana-labs/rbpf/blob/302de4d10888dfb76534693308f4ef8060b86fcd/src/vm.rs#L206
Should we disable it in the next release?

Reply:
`instruction_meter_checkpoint_distance` should have no influence on what the program sees, the point where a programs is aborted and what error is throws has to be the same for interpreter and JIT anyway. The option only makes the detection of that condition faster (so it affects the performance only).

Reply:
Sounds good as long as there isn't an edge case that's falls in the middle.  Also, maybe check with Tao about cost model stuff to make sure this change doesn't result in a behavioral change there.